src/main/java/com/tvtracker/enterprise/dao/MediaEntryDAO.java

```diff
@@ -37,8 +29,8 @@ public boolean save(MediaEntry mediaEntry) throws SQLException, IOException, Cla
      */
     @Override
     public List<MediaEntry> fetchByUsername(String username) throws SQLException, IOException, ClassNotFoundException {
-        addWhere("username", username);
-        return parse(select());
+        List<MediaEntry> returnMediaEntries = new ArrayList(Collections.singleton(allMediaEntries.containsValue(username)));
```

Comment:
There's a lot going on, in this one line.  If an exception occurs here, it can be hard to figure out where (on which method call) the exception occurred.  Best to have one call per line.